src/TCProcessor.cpp

```diff
@@ -236,7 +270,12 @@ TCProcessor::create_decision(const PendingTD& pending_td)
       decision.trigger_type = m_trigger_type_shifted;
     }
   } else {
-    decision.trigger_type = 1; // m_trigger_type;
+    m_TD_bitword = get_TD_bitword(pending_td);
+    TLOG() << "[MLT] TD has bitword: " << m_TD_bitword << " "
```

Comment:
```suggestion
    TLOG_DEBUG(5) << "[MLT] TD has bitword: " << m_TD_bitword << " "
```
I think we want this as a debugging message, rather than always in the log?